bindings/java/src/main/com/apple/foundationdb/tuple/FastByteComparisons.java

```diff
@@ -30,22 +30,22 @@
 
 /**
  * Utility code to do optimized byte-array comparison.
- * This is borrowed and slightly modified from Guava's {@link UnsignedBytes}
+ * This is borrowed and slightly modified from Guava's  UnsignedBytes
  * class to be able to compare arrays that start at non-zero offsets.
  */
 abstract class FastByteComparisons {
 
     private static final int UNSIGNED_MASK = 0xFF;
     /**
      * Lexicographically compare two byte arrays.
-     * 
+     *
      * @param buffer1 left operand, expected to not be null
      * @param buffer2 right operand, expected to not be null
-     * @param offset1 Where to start comparing in the left buffer, expected to be >= 0
-     * @param offset2 Where to start comparing in the right buffer, expected to be >= 0
-     * @param length1 How much to compare from the left buffer, expected to be >= 0
-     * @param length2 How much to compare from the right buffer, expected to be >= 0
-     * @return 0 if equal, < 0 if left is less than right, etc.
+     * @param offset1 Where to start comparing in the left buffer, expected to be &rt;= 0
```

Comment:
Does this work? Usually I see this written as `&gt;` instead.

Reply:
This doesn't work as expected but it also didn't produce an error. It was a typo.